src/features/parish/pages/parish-generic-page/index.tsx

```
@@ -21,7 +24,24 @@ const ParishGenericPage = () => {
 				activeTabId={activeIndex}
 				tabs={tabsData || [{ label: 'view' }, { label: 'add' }]}
 			>
-				<h1 className="text-center text-red-600">dynamic table will be added ...</h1>
+				<div className="">
+					{['parish_history', 'patron_saint'].includes(pathName as string) && (
+						<>{activeIndex == 0 ? <DisplayImage image="" /> : <HistoryForm />} </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)